owsi-core/owsi-core-examples/wicket-showcase/wicket-showcase-webapp/src/main/java/fr/openwide/core/showcase/web/application/config/spring/ShowcaseWebappApplicationPropertyConfig.java

```diff
@@ -0,0 +1,18 @@
+package fr.openwide.core.showcase.web.application.config.spring;
+
+import org.springframework.context.annotation.Configuration;
+
+import fr.openwide.core.jpa.more.business.property.service.IConfigurablePropertyService;
+import fr.openwide.core.jpa.more.config.spring.AbstractApplicationPropertyConfig;
+import fr.openwide.core.showcase.web.application.util.property.ShowcaseWebappPropertyIds;
+
+@Configuration
+public class ShowcaseWebappApplicationPropertyConfig extends AbstractApplicationPropertyConfig {
```

Comment:
My bad, I didn't think about the singleton beans. As you suggested, I added <code>AbstractApplicationPropertyRegisterConfig</code> to register propreties and <code>AbstractApplicationPropertyConfig</code> extends the register config and defines the beans. Thanks!